resources/metadata/sonarExecuteScan.yaml

```diff
@@ -296,4 +296,4 @@ spec:
                 type: int
   containers:
     - name: sonar
-      image: sonarsource/sonar-scanner-cli:4.6
+      image: sonarsource/sonar-scanner-cli:4.7
```

Comment:
Release happened a while ago : 
https://github.com/SonarSource/sonar-scanner-cli/releases/tag/4.7.0.2747
Why this is relevant now ?

Reply:
sonarExecuteScan fails and we are trying to fix the issue.
A colleague from the Sonar team mentioned that «The default version of piper configuration should use sonar scaner version 4.7. It is the correct version which should be used»

Reply:
@I501307 
teams are blocked and we are trying to fix as soon as possible.
Do you see any issue with upgrading the sonar image? If not can we try merging this please?

Reply:
Thank you for creating this PR @vstarostin. We have to keep in mind following points: 

1. As discussed on teams, could you please establish the fact that updating sonarcli dockerImage to v4.7 will actually solve your problem with error : 
`Exception caught during execution of command '[/usr/bin/git, config, --system, --show-origin, --list, -z]' in '/usr/bin', return code '128', error message 'fatal: unable to read config file '/etc/gitconfig': No such file or directory`
2. We have to update the dockerImage to v4.7 regardless of the above error. However, this image must be throughly tested and scanned for any vulnerabilities. I will take care of this.
3. Once we have tested/scanned it - you will have green light here 👍